core/shared/src/main/scala/spire/algebra/Homomorphism.scala

```diff
@@ -0,0 +1,54 @@
+package spire.algebra
+
+import java.math.BigInteger
+
+import spire.math.{Complex, SafeLong}
+
+trait Homomorphism[-A, +B, +F[_]] extends Function1[A, B]
```

Comment:
@tixxit, homomorphisms do not extend `Function1` anymore. Instead, their implementation *could*, and they provide a `.toFunction` method.